dto/company.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsNotEmpty } from "class-validator";
+
+
+export class CompanyDTO {
+
+    @IsNotEmpty({ message: 'Company name is required' })

```

Comment:
there should be more validations to validate the company name i.e min and max length should be there, no special characters should be allowed and many other.